package.json

```diff
@@ -60,9 +60,9 @@
     "typescript": "~3.5.3"
   },
   "husky": {
-  "hooks": {
-    "pre-commit": "ng lint",
-    "pre-push":"ng test --no-watch"
+    "hooks": {
```

Comment:
Hey @Dhamodhar-DDR why do we need this change in here? Please revert this change

Reply:
I dont think there is any change here, i just added a tab space to those three lines.